src/data/constant.tsx

```diff
@@ -53,7 +54,19 @@ class dataInLocalStorage {
   }
 }
 
-export const token = new dataInLocalStorage('token');
-console.log('token=', token);
+export const Token = new dataInLocalStorage('token');
+export const Email = new dataInLocalStorage('email');
+export const Password = new dataInLocalStorage('password');
 
 export {memoize};
+
+export const tokenSet = (token:string):void => {
+  localStorage.setItem('token',token);
+  axios.defaults.headers['x-token'] = token;
+};
+
+export const tokenGet = () => localStorage.getItem('token');
+
+export const tokenDelete = (token:string):void => {
+  localStorage.removeItem('token');
```

Comment:
обычно глаголы в названиях в начале пишут
```suggestion
export const setToken = (token:string):void => {
  localStorage.setItem('token',token);
  axios.defaults.headers['x-token'] = token;
};

export const getToken = () => localStorage.getItem('token');

export const deleteToken = (token:string):void => {
  localStorage.removeItem('token');
```